### PR TITLE
Add order options by name.

### PR DESCRIPTION
**Pull request by vukvukovich**
https://onthegosystems.myjetbrains.com/youtrack/issue/mt-67

### CODE REVIEW

**Comment by kkarpieszuk** [comment]
@vukvukovich I see your `function wpml_ctt_load_alloptions()` is almost the same as wp core function `function wp_load_alloptions()` except your are sorting results in alphabetical order. To not repeat the code you should use function `wp_load_alloptions()` and do `asort()` on the results and that's all.

**Comment by vukvukovich** [comment]
> @vukvukovich I see your `function wpml_ctt_load_alloptions()` is almost the same as wp core function `function wp_load_alloptions()` except your are sorting results in alphabetical order. To not repeat the code you should use function `wp_load_alloptions()` and do `asort()` on the results and that's all.

This function need to be there since original one was limiting our options with `WHERE autoload = 'yes'`.

**Comment by kkarpieszuk** [comment]
right. Then I have nothing more to add. Approved

**Comment by vukvukovich** [comment]
> right. Then I have nothing more to add. Approved

@kkarpieszuk `asort()` looks like overhead in this situation don't you think?

**Comment by dgwatkins** [comment]
@vukvukovich please resolve conflicts so I can test this.

File: inc/wpml-compatibility-test-tools.functions.php [hunk 1]
```diff
@@ -319,29 +319,31 @@ function wpml_ctt_validate_radio( $value ) {
...
  */
 function wpml_ctt_load_alloptions() {
-    global $wpdb;
+	global $wpdb;
 
-    if ( ! wp_installing() || ! is_multisite() )
-        $alloptions = wp_cache_get( 'wpml_ctt_all_options', 'options' );
-    else
-        $alloptions = false;
+	if ( ! wp_installing() || ! is_multisite() ) {
```
Comment (dgwatkins) [inline inc/wpml-compatibility-test-tools.functions.php:324]: @vukvukovich 
When do you want to use cache here?
In other words, can you explain this `if` in human language?

Is it: "when we are installing on multisite dont use cache"
or is it: "when we are installing or on a multisite dont use cache"
which is the correct one?

**Comment by vukvukovich** [comment]
> @vukvukovich please resolve conflicts so I can test this.

@dgwatkins Done, please check it out.

**Comment by dgwatkins** [comment]
I tested and it works.

I opened a review above, the rest is good.

File: inc/wpml-compatibility-test-tools.functions.php (hunk 1 above)
Comment (vukvukovich) [reply inc/wpml-compatibility-test-tools.functions.php:324]: @dgwatkins I just took `wp_load_alloptions()` original function content and removed part regarding autoload. I've left other code same.

Comment (dgwatkins) [reply inc/wpml-compatibility-test-tools.functions.php:324]: ok, then it must be right :+1:

**Review by dgwatkins — APPROVED** [review]

**Pull request merged**